vmdb/app/models/miq_report/search.rb

```diff
@@ -33,7 +33,7 @@ def get_cached_page(limit, offset, includes, options)
 
   def get_order_info
     order = nil
-    apply_sortby_in_search = !self.db_class.table_name.nil?
+    apply_sortby_in_search = self.db_class.reportable?
```

Comment:
@tenderlove I love the idea of asking the class... but I don't think the wording is right.  These are reportable, right?  ActsAsArModel is used in reporting, that's it purpose.  I think `sortable?` might be closer to what this code is doing.